### ISSUE TITLE
Fails to install dependencies

### ISSUE THREAD

**Issue by olwe1** [janoside/btc-rpc-explorer#161]
**Describe the bug**

When `npm install -g btc-rpc-explorer` or clone the project and doing `npm install` it fails.

I had the same error on both raspbian and macOs.

**Environment (please complete the following information):**

 - NodeJS Version [e.g. 9.x] : v12.12.0
- NPM Version : 6.13.1
 - Code Version / Commit [e.g. ab6cde8] : [https://github.com/janoside/btc-rpc-explorer/commit/195874987191fd68c50dcba967d8833d992e5b60](195874987191fd68c50dcba967d8833d992e5b60)

**To Reproduce**

1. `git clone https://github.com/janoside/btc-rpc-explorer.git`
2. `npm i`

**Screenshots or Log Output**

```
npm i                 

> dtrace-provider@0.8.7 install /Users/****/Desktop/explorer/btc-rpc-explorer/node_modules/dtrace-provider
> node-gyp rebuild || node suppress-error.js

  ACTION binding_gyp_ndtp_target_build_ndtp .
-----------------------------------------------------------------
Building dtrace-provider failed with exit code 1, falling back
on stub implementation.

Re-run install with V set in your environment to see the build
output, or NODE_DTRACE_PROVIDER_REQUIRE=hard to force an
installation failure.
-----------------------------------------------------------------
  TOUCH Release/obj.target/ndtp.stamp

> tiny-secp256k1@1.1.0 install /Users/****/Desktop/explorer/btc-rpc-explorer/node_modules/tiny-secp256k1
> node-gyp rebuild

  CXX(target) Release/obj.target/secp256k1/native/addon.o
../native/addon.cpp:240:2: warning: ignoring return value of function declared with 'warn_unused_result' attribute [-Wunused-result]
        secp256k1_ec_privkey_negate(context, tweak_negated); // returns 1 always
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~
../native/addon.cpp:320:21: error: no matching member function for call to 'BooleanValue'
                strict = info[3]->BooleanValue();
                         ~~~~~~~~~^~~~~~~~~~~~
/Users/****/Library/Caches/node-gyp/12.12.0/include/node/v8.h:2603:8: note: candidate function not viable: requires single argument
      'isolate', but no arguments were provided
  bool BooleanValue(Isolate* isolate) const;
       ^
/Users/****/Library/Caches/node-gyp/12.12.0/include/node/v8.h:2606:51: note: candidate function not viable: requires single argument
      'context', but no arguments were provided
                V8_WARN_UNUSED_RESULT Maybe<bool> BooleanValue(
                                                  ^
../native/addon.cpp:82:23: error: no matching member function for call to 'BooleanValue'
                return info[index]->BooleanValue() ? SECP256K1_EC_COMPRESSED : SECP256K1_EC_UNCOMPRESSED;
                       ~~~~~~~~~~~~~^~~~~~~~~~~~
../native/addon.cpp:142:21: note: in instantiation of function template specialization '(anonymous namespace)::assumeCompression<2,
      Nan::FunctionCallbackInfo<v8::Value>, v8::Local<v8::Object> >' requested here
        const auto flags = assumeCompression<2>(info, pA);
                           ^
/Users/****/Library/Caches/node-gyp/12.12.0/include/node/v8.h:2603:8: note: candidate function not viable: requires single argument
      'isolate', but no arguments were provided
  bool BooleanValue(Isolate* isolate) const;
       ^
/Users/****/Library/Caches/node-gyp/12.12.0/include/node/v8.h:2606:51: note: candidate function not viable: requires single argument
      'context', but no arguments were provided
                V8_WARN_UNUSED_RESULT Maybe<bool> BooleanValue(
                                                  ^
../native/addon.cpp:80:21: warning: comparison of integers of different signs: 'int' and 'unsigned long' [-Wsign-compare]
                if (info.Length() <= index) return __isPointCompressed(p) ? SECP256K1_EC_COMPRESSED : SECP256K1_EC_UNCOMPRESSED;
                    ~~~~~~~~~~~~~ ^  ~~~~~
../native/addon.cpp:82:23: error: no matching member function for call to 'BooleanValue'
                return info[index]->BooleanValue() ? SECP256K1_EC_COMPRESSED : SECP256K1_EC_UNCOMPRESSED;
                       ~~~~~~~~~~~~~^~~~~~~~~~~~
../native/addon.cpp:174:21: note: in instantiation of function template specialization '(anonymous namespace)::assumeCompression<1,
      Nan::FunctionCallbackInfo<v8::Value>, v8::Local<v8::Object> >' requested here
        const auto flags = assumeCompression<1>(info, p);
                           ^
/Users/****/Library/Caches/node-gyp/12.12.0/include/node/v8.h:2603:8: note: candidate function not viable: requires single argument
      'isolate', but no arguments were provided
  bool BooleanValue(Isolate* isolate) const;
       ^
/Users/****/Library/Caches/node-gyp/12.12.0/include/node/v8.h:2606:51: note: candidate function not viable: requires single argument
      'context', but no arguments were provided
                V8_WARN_UNUSED_RESULT Maybe<bool> BooleanValue(
                                                  ^
../native/addon.cpp:80:21: warning: comparison of integers of different signs: 'int' and 'unsigned long' [-Wsign-compare]
                if (info.Length() <= index) return __isPointCompressed(p) ? SECP256K1_EC_COMPRESSED : SECP256K1_EC_UNCOMPRESSED;
                    ~~~~~~~~~~~~~ ^  ~~~~~
../native/addon.cpp:89:23: error: no matching member function for call to 'BooleanValue'
                return info[index]->BooleanValue() ? SECP256K1_EC_COMPRESSED : SECP256K1_EC_UNCOMPRESSED;
                       ~~~~~~~~~~~~~^~~~~~~~~~~~
../native/addon.cpp:189:21: note: in instantiation of function template specialization '(anonymous namespace)::assumeCompression<1,
      Nan::FunctionCallbackInfo<v8::Value> >' requested here
        const auto flags = assumeCompression<1>(info);
                           ^
/Users/****/Library/Caches/node-gyp/12.12.0/include/node/v8.h:2603:8: note: candidate function not viable: requires single argument
      'isolate', but no arguments were provided
  bool BooleanValue(Isolate* isolate) const;
       ^
/Users/****/Library/Caches/node-gyp/12.12.0/include/node/v8.h:2606:51: note: candidate function not viable: requires single argument
      'context', but no arguments were provided
                V8_WARN_UNUSED_RESULT Maybe<bool> BooleanValue(
                                                  ^
../native/addon.cpp:87:21: warning: comparison of integers of different signs: 'int' and 'unsigned long' [-Wsign-compare]
                if (info.Length() <= index) return SECP256K1_EC_COMPRESSED;
                    ~~~~~~~~~~~~~ ^  ~~~~~
4 warnings and 4 errors generated.
make: *** [Release/obj.target/secp256k1/native/addon.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 19.0.0
gyp ERR! command "/usr/local/Cellar/node/12.12.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/****/Desktop/explorer/btc-rpc-explorer/node_modules/tiny-secp256k1
gyp ERR! node -v v12.12.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tiny-secp256k1@1.1.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the tiny-secp256k1@1.1.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/****/.npm/_logs/2019-11-29T14_54_42_628Z-debug.log
```

**Comment by janoside** [comment]
@olwe1 I just published a new version: v1.1.3 that includes updated dependency versions. When you get a chance, please let me know if this error is fixed in the new version.

**Comment by olwe1** [comment]
It's working fine
Thank you for your work !
